lib/Watcher.php

```diff
@@ -116,6 +117,14 @@ public function postWrite(Node $node) {
 			return;
 		}
 
+		// NOTE: The hooks also report the creation of thumbnails, and other files.
+		// The way to differentiate between them, is that the user files starting with the userId, and the thumbains as appdata_ocs######
+		$absPath = ltrim($node->getPath(), '/');
+		$owner = explode('/', $absPath)[0];
+		if (!$this->userManager->userExists($owner)) {
```

Comment:
this check (that user exists) is OK for local/home files. Not sure how will this look like with shared/external files?